README.md

```diff
@@ -0,0 +1,247 @@
+
+- Run the application now by typing the `appsody run` command.
+
+- Navigate to `http://localhost:3000` to see the output.
+    **NOTE**: The URL can be different, depending on the stack, so consult the documentation.
+
+- Edit `app.js` so that it outputs something other than "Hello from Appsody!". When you save the file, the change is detected and the container is automatically updated.
+
+- Refresh `http://localhost:3000` to see the new message.
+
+### Checking the status of your running container
+
+**CLI command:** `appsody ps`
+
+To list all the stack-based containers that are running in your local environment, use the `appsody ps` command. The output provides information about the container ID, name, image, and the status of each container.
+
+- Run the `appsody ps` command to see output that is similar to the following example:
+
+```
+CONTAINER ID	NAME            IMAGE                       	STATUS
+f20ec098a612	my-project-dev	abc-stacks/nodejs-express:0.2	Up 8 minutes
+```
+
+### Stopping your Appsody container
+
+**CLI command:** `appsody stop --name <container-name>`
+
+To stop a container you can either press `Ctrl-C` in the terminal or use the `appsody stop` command.
+
+If you have more than one development project open, use the `appsody stop --name <container-name>` to stop a specific container. Use the `appsody ps` command to find the name of the container you want to stop.
+
+## Testing your application
+
+**CLI command:** `appsody test`
+
+The `appsody test` command runs the test suite for your application in the development container. Each application stack provides a set of generic tests, which verify that the capabilities provided by the stack are working as expected. Typically, these tests check that the endpoints that are created, such as `/metrics` and `/health`, are available. In addition, you can define further tests for your application in your project `/test` folder.
+
+In earlier sections of this guide you created a `nodejs-express` project with the default (`simple`) template, which provides a sample test as a starting point. Take a look at the sample test in the `my-project/test/test.js` file. You can update this file to suit your test requirements.
+
+Now try running the `appsody test` command for your project. The results from the test suite are included in the output.
+
+The testing uses constructs that are familiar to the programming language or framework on which the stack is based. You can add your own tests or switch to your preferred testing framework. Node.js application stacks use the [Mocha](https://mochajs.org/) test framework as default. If you want to use a different test framework, update the `npm test` command in your project `package.json` file.
```

Comment:
where do you do the `npm test` upgrade?  the application stack container, or is this the local environment?

Reply:
If you update the `package.json` file the changes apply both to a local development container and the deployment container.  Do you think this needs to be called out specifically @stephenkinder ?

Reply:
@SueChaplain I'll leave it to you as to whether it belongs here, elsewhere or nowhere.